package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-tools",
-  "version": "1.7.4",
+  "version": "1.7.5",
```

Comment:
since this is an API change we need it in 1.8.0 scheduled to be released in 2 weeks. Please create a 1.8 branch under master. Any subsequent API changes will be made there.

Reply:
Do you mean to create branch from this master, not my fork? Do I have appropriate rights?